include/itkPhaseCorrelationImageRegistrationMethod.h

```diff
@@ -478,10 +478,10 @@ class ITK_TEMPLATE_EXPORT PhaseCorrelationImageRegistrationMethod : public Proce
   typename ComplexImageType::Pointer m_FixedImageFFT = nullptr;
   typename ComplexImageType::Pointer m_MovingImageFFT = nullptr;
 
-  ParametersType m_TransformParameters;
-  SizeType       m_PadToSize;
-  SizeType       m_ObligatoryPadding;
-  PaddingMethod  m_PaddingMethod = PaddingMethod::MirrorWithExponentialDecay;
+  ParametersType                       m_TransformParameters;
+  SizeType                             m_PadToSize;
+  SizeType                             m_ObligatoryPadding;
```

Comment:
Is this alignment correct? Seems weird to me.

Reply:
I ran clang-format, so all style is conformant :nail_care: